integrationTests/frontend/wallet/txInterception_test.go

```diff
@@ -121,16 +101,17 @@ func testInterceptedTxFromFrontendGeneratedParams(
 
 	integrationTests.MintAddress(node.AccntState, sndAddrBytes, valMinting)
 
-	txHexHash, err = node.SendTransaction(&transaction.Transaction{
+	tx := &transaction.Transaction{
```

Comment:
You can put back: Value:     frontendValue and remove tx.SetValue(frontendValue)